packages/devtools_app/lib/src/snapshot_screen.dart

```diff
@@ -69,7 +70,8 @@ class _SnapshotScreenBodyState extends State<SnapshotScreenBody> {
         Row(
           children: [
             ExitOfflineButton(onPressed: () {
-              Navigator.pop(context);
+              DevToolsRouterDelegate.of(context)
+                  .pushPageIfNotCurrent(homePageId, {'screen': null});
```

Comment:
With this change, is the snapshot screen still on the navigation stack? I'm not if sure we want a user to be able to go "back" to the offline snapshot they just cleared.

Reply:
I've changed this to the new `replacePage` as above (though it has the same back button issue that needs fixing).